doc/admin/deployment.md

```diff
@@ -594,7 +594,7 @@ Typically an administrator will perform the following tasks:
     - just specify NVMe PCI addresses with `bdev_list` for now
 
 4. Start DAOS control plane
-    - `orterun -np 2 -H boro-44,boro-45 --enable-recovery daos_server -a shared_dir -o <daos>/utils/config/examples/daos_server_sockets.yml start`
+    - `orterun -np 2 -H boro-44,boro-45 --enable-recovery daos_server start -a shared_dir -o <daos>/utils/config/examples/daos_server_sockets.yml`
```

Comment:
The --config, -o flag is a global option and should come before the subcommand. I believe that this invocation would cause an error after your patch lands.

Reply:
no they are order agnostic

Reply:
Hmm, interesting. I guess I'm misremembering. Perhaps what I was seeing was a conflict between global/command options when I was working on this before. Anyhow, I just pulled this PR to test and can confirm that supplying -o/--config after start works fine.